internal/website/docs/next/reference/hooks/usePage.mdx

```diff
@@ -46,9 +46,9 @@ import { client, PageIdType } from 'client';
 
 export default function Page() {
   const { usePost } = client;
-  const page = usePage({
+  const page = usePost({
```

Comment:
Shouldn't this be `usePage` instead of `usePost`? The [original diff](https://github.com/wpengine/faustjs/pull/502/files) looked slightly different.